NEWS.md

```diff
@@ -21,6 +21,8 @@
 
 ## Minor changes and fixes
 
+- `style_dir()` and friends now apply directory exclusion recursively with 
```

Comment:
Shouldn't we be explicit since the only other function that currently does that is `style_pkg?`

>  `style_dir()` and `style_pkg()` now apply directory exclusion recursively with

Reply:
I agree. Did yoy know about the 'Insert Suggestion' tool? You can directly suggest a new commit on GitHub.
![Screenshot 2020-10-04 at 10 35 42](https://user-images.githubusercontent.com/10477073/95010966-7e77af80-062d-11eb-8a3b-735556f78107.png)

Reply:
Didn't know about it. Thanks for sharing!